raiden/utils/__init__.py

```diff
@@ -160,3 +160,33 @@ def camel_to_snake_case(name):
 
 def snake_to_camel_case(snake_string):
     return snake_string.title().replace("_", "")
+
+
+def bytes_to_hexstr(b):
+    return '0x' + b.encode('hex')
+
+
+def netting_channel_to_api_dict(nettingchannel, our_address):
+    """Takes in a NettingChannel Object and turns it into a dictionary for
+    usage in the REST API.
+
+    Note: `our_address` is required since it's not part of either the mock or
+          tester implementation of NettingChannel. Only the rpc NettingChannel
+          knows its node address.
+    """
+
+    state = 'open'
```

Comment:
`isopen`, `settled`, `detail`, and `token_address` are all proxies for the netting channel's functions that are **not** cached, for this reason this function will be slow and do too many json-rpc requests, using the Raiden's channel representation is a better alternative.